xbmc/addons/interfaces/Filesystem.cpp

```diff
@@ -503,6 +504,40 @@ int Interface_Filesystem::get_file_chunk_size(void* kodiBase, void* file)
   return static_cast<CFile*>(file)->GetChunkSize();
 }
 
+const char* Interface_Filesystem::get_property(void* kodiBase, void* file, int type, const char *name)
+{
+  CAddonDll* addon = static_cast<CAddonDll*>(kodiBase);
+  if (addon == nullptr || file == nullptr)
```

Comment:
Can name be null to?

Reply:
No, because the Interface class methods all work with std::string
const char* is only used for the step dll <-> kodi

Reply:
I would add the check nevertheless. It does not cost to much computing power and makes the code more failsafe.